### ISSUE TITLE
Failure to Initialize!

### ISSUE THREAD

**Issue by starterbeeee** [MustardChef/MagiskOnGPGPC#6]
![imagen](https://github.com/MustardChef/MagiskOnGPGPC/assets/83183177/fa3b2382-786e-454e-869b-9c5c8d523b41)
Not sure what to do - i have followed the steps in the release section.
The application showing this error is mario run, the armeabi-v7a version.

**Comment by MustardChef** [comment]
Thank you for reporting, I will have a look.

**Comment by starterbeeee** [comment]
Hi, i tried a couple more apps too - but i have gotten no ARM apps to work.
One thing i noticed, after flashing Houdini more apps that are only ARM appeared out of nowhere available for me to download on the GPG app.
![imagen](https://github.com/MustardChef/MagiskOnGPGPC/assets/83183177/8e8a0737-dd71-4dc4-9b8c-c7ffa8d27bd9)
They'll install fine, but opening them ends up in a crash.
Most just quit without a error, others crash with a error like Mario Run.
Did i do something wrong? I flashed magisk and other modules worked fine, root works fine as well, and i also turned Zygisk on.
I flashed the OverlayFS module too.

**Comment by MustardChef** [comment]
I have found the problem causing the issue. Wait for the update. 
Thank you for the extra information.

**Comment by admin-elvistrujanovic** [comment]
> I have found the problem causing the issue. Wait for the update. 
> Thank you for the extra information.

Btw, on developer edition, there is already built-in libhoudini (and Android 15 I saw) but it didn't worked with this module - it crashed all arm-based apps

Also, about bootloader bypass, when I've installed Magisk and opened the app, it asked me that Magisk needs to do additional settings. When I clicked ok, it rebooted emulator and when I've opened it, the Magisk responded that I need to re-flash the Magisk using the app since something is wrong (It cannot read device info and etc by recovery and similar warnings...)


When I've tried to use LSPosed forked that works for Android 15, it showed me the warning in LSPosed that Magisk is not properly installed, it cannot use SELinux and most features will not work.


What you think - what could cause the issue. I've tried also different Magisk distribution too like Kitsune Mask but it didn't worked too. Could it be due to the bypass? Maybe your bypass method will actually be more universal (better for compatibility when it comes to these things and re-flashing the Magisk using the app in the emulator without any limitations on it)


But I'm sure that this bypass would work better with Android 14 (that is currently stable latest version for Google Play Games Beta - PC edition)

**Comment by MustardChef** [comment]
> > I have found the problem causing the issue. Wait for the update.
> > Thank you for the extra information.
> 
> Btw, on developer edition, there is already built-in libhoudini (and Android 15 I saw) but it didn't worked with this module - it crashed all arm-based apps
> 
> Also, about bootloader bypass, when I've installed Magisk and opened the app, it asked me that Magisk needs to do additional settings. When I clicked ok, it rebooted emulator and when I've opened it, the Magisk responded that I need to re-flash the Magisk using the app since something is wrong (It cannot read device info and etc by recovery and similar warnings...)
> 
> When I've tried to use LSPosed forked that works for Android 15, it showed me the warning in LSPosed that Magisk is not properly installed, it cannot use SELinux and most features will not work.
> 
> What you think - what could cause the issue. I've tried also different Magisk distribution too like Kitsune Mask but it didn't worked too. Could it be due to the bypass? Maybe your bypass method will actually be more universal (better for compatibility when it comes to these things and re-flashing the Magisk using the app in the emulator without any limitations on it)
> 
> But I'm sure that this bypass would work better with Android 14 (that is currently stable latest version for Google Play Games Beta - PC edition)

Agreed, the emulator already comes with Native Bridge, but you will have to enable it.
